crates/client/src/lib.rs

```diff
@@ -14,12 +14,11 @@
    limitations under the License.
 */
 
+#![cfg_attr(feature = "docs", doc = include_str!("../README.md"))]
```

Comment:
does this need the `//!`` so it formats properly?

Reply:
No, we can include markdown directly here.
You can verify the final result with:
```bash
cargo doc --no-deps --features docs --open
```

Reply:
cool!